index.html

```diff
@@ -953,7 +958,7 @@ <h4>createDisclosureData</h4>
             </li>
             <li>
 Return an object with properties matching |bbsProof|, "verifierLabelMap" for |labelMap|,
-|mandatoryIndexes|, |selectiveIndexes|, |revealDocument|, and |pseudonym|, if
+|mandatoryIndexes|, |selectiveIndexes|, |revealDocument|, |pseudonym|, and |L| if
```

Comment:
We should use a better semantic name than `L` even if we want to say that it maps to `L` in a low-level BBS spec. The letter `L` itself can be confused with other symbols -- but at least this is less likely with an upper-case `L`.

Reply:
Hi @dlongley, *L* is the number of issuer signed BBS messages. Or in W3C VC-DI-BBS speak the number of non-mandatory n-quads. When working with Blind BBS and Pseudonym BBS *proofs* (W3C derived proofs) this is a new parameter that is needed during verification since we separated out the "group generators" for the blinded (holder) messages from the issuer messages to simplify indexing and to unify pseudonym proof verification. I'll take any suggestions for naming!